modules/serverless-rn-1-16-0.adoc

```diff
@@ -0,0 +1,116 @@
+[id="serverless-rn-1-16-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.16.0
+
+[id="new-features-1-16-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.22.0.
+* {ServerlessProductName} now uses Knative Eventing 0.22.0.
+* {ServerlessProductName} now uses Kourier 0.22.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.22.0.
+* {ServerlessProductName} now uses Knative Kafka 0.22.0.
+* The `kn func` CLI plugin now uses `func` 0.16.0.
```

Comment:
```suggestion
* The `kn func` CLI plug-in now uses `func` 0.16.0.
```

> 🚫  plugin noun, adjective
Do not use. Use "plug-in".

I see at least one other instance to update, but please fix them all.